packages/office-ui-fabric-react/src/components/Dialog/Dialog.scss

```diff
@@ -17,209 +17,197 @@ $Dialog-default-max-width: 340px;
-    flex-direction: row;
-    flex-wrap: nowrap;
-    position: absolute;
-    top: 0;
-    @include right(0);
-    @include padding(12px, 12px, 0, 0);
-    > * {
-      flex: 0 0 auto;
-    }
+.title {
+  margin: 0;
+  @include ms-font-xl;
+  color: $ms-color-neutralPrimary;
+  @include padding(20px, 36px, 20px, 28px);
+}
+
+.topButton {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  position: absolute;
+  top: 0;
+  @include right(0);
+  @include padding(12px, 12px, 0, 0);
+  > * {
+    flex: 0 0 auto;
   }
+}
 
-  .ms-Dialog-content {
-    position: relative;
-    width: 100%;
+.content {
+  position: relative;
+  width: 100%;
 
-    // Add margin bottom between compound buttons
-    .ms-Button.ms-Button--compound {
-      margin-bottom: 20px;
+  // Add margin bottom between compound buttons
+  :global(.ms-Button.ms-Button--compound) {
+    margin-bottom: 20px;
 
-      &:last-child {
-        margin-bottom: 0;
-      }
+    &:last-child {
+      margin-bottom: 0;
     }
   }
+}
 
-  .ms-Dialog-subText {
-    margin: 0 0 20px 0;
-    padding-top: 8px;
-    @include ms-font-s;
-    color: $ms-color-neutralPrimary;
-    font-weight: $ms-font-weight-semilight;
-    line-height: 1.5;
-  }
+.subText {
+  margin: 0 0 20px 0;
+  padding-top: 8px;
+  @include ms-font-s;
+  color: $ms-color-neutralPrimary;
+  font-weight: $ms-font-weight-semilight;
+  line-height: 1.5;
+}
 
-  .ms-Dialog-actions {
-    position: relative;
-    width: 100%;
-    min-height:  24px;
-    line-height: 24px;
-    margin: 20px 0 0;
-    font-size: 0;
+.actions {
+  position: relative;
+  width: 100%;
+  min-height:  24px;
+  line-height: 24px;
+  margin: 20px 0 0;
+  font-size: 0;
 
-    .ms-Button {
-      line-height: normal;
-    }
+  :global(.ms-Button) {
+    line-height: normal;
   }
+}
 
-  // Negative margin to needed to compensate for symmetric
-  // padding between action elements.
-  .ms-Dialog-actionsRight {
-    @include text-align(right);
-    @include margin-right(-4px);
-    font-size: 0;
+// Negative margin to needed to compensate for symmetric
+// padding between action elements.
+.actionsRight {
+  @include text-align(right);
+  @include margin-right(-4px);
+  font-size: 0;
 
-    // Reset spacing for first button
-    .ms-Dialog-action {
-      margin: 0px 4px;
-    }
+  // Reset spacing for first button
+  .action {
+    margin: 0px 4px;
   }
+}
 
-  //= Modifier: Dialog with close button
-  //
-  .ms-Dialog.ms-Dialog--close:not(.ms-Dialog--lgHeader) {
-    // Show the close button
-    .ms-Dialog-button.ms-Dialog-button--close {
-      display: block;
-    }
+//= Modifier: Dialog with close button
+//
+.root.isClose:not(.isLargeHeader) {
+  // Show the close button
+  .button.isClose {
+    display: block;
   }
+}
 
-  //= Modifier: Multiline button dialog
-  //
-  .ms-Dialog.ms-Dialog--multiline {
-    .ms-Dialog-title {
-      font-size: $ms-font-size-xxl;
-    }
+//= Modifier: Multiline button dialog
+//
+.root.isMultiline {
```

Comment:
`.rootIsMultiline`, unless we want to wait and do the 'flattening' in a separate PR.